ros2pkg/test/test_cli.py

```diff
@@ -200,6 +201,6 @@ def test_create_package(self):
             assert root.find('description').text == 'A test package dummy description'
             assert root.find('maintainer').text == 'Nobody'
             assert root.find('maintainer').attrib['email'] == 'nobody@nowhere.com'
-            assert root.find('license').text == 'Apache License 2.0'
+            assert root.find('license').text == 'Apache License, Version 2.0'
```

Comment:
What I've seen (and assumed it was the SPDX identifier) was `Apache-2.0`. I guess there are different styles or something?

Reply:
No, I'm just dumb.  I never re-ran the tests after I switched to SPDX identifiers, so this test is just going to fail :).  Thanks for pointing it out, I'll update it.